mmdet/models/roi_heads/mask_heads/coarse_mask_head.py

```diff
@@ -0,0 +1,78 @@
+import torch.nn as nn
+from mmcv.cnn import ConvModule, constant_init, xavier_init
+
+from mmdet.core import auto_fp16
+from mmdet.models.builder import HEADS
+from .fcn_mask_head import FCNMaskHead
+
+
+@HEADS.register_module
+class CoarseMaskHead(FCNMaskHead):
+
```

Comment:
Docstring, describe the main difference between `CoarseMaskHead` and `FCNMaskHead` and its usage.